app/views/index.scala.html

```diff
@@ -1,4 +1,4 @@
-@(person: Form[CreatePersonForm])(implicit messages: Messages)
+@(person: Form[CreatePersonForm])(implicit reqeust: RequestHeader, messages: Messages)
```

Comment:
You misspelled, "request", but thank you for the PR, I updated my code locally to match this.